src/components/collapsible-section/collapsible-section.tsx

```diff
@@ -92,6 +108,18 @@ export class CollapsibleSection {
         }
     }
 
+    private handleKeyDown = (event: KeyboardEvent) => {
+        const isEnter = event.key === ENTER || event.keyCode === ENTER_KEY_CODE;
+
+        if (isEnter) {
+            event.stopPropagation();
+            event.preventDefault();
+            const element = (this.host.shadowRoot.activeElement ||
+                document.activeElement) as HTMLElement;
+            element.click();
```

Comment:
@Kiarokh @jensgustafsson For next time, I think it would be better to refactor out the stuff that happens in `onClick` into a common function that could be called by both `onClick` and `handleKeyDown`, or even just outright call `onClick`, rather than synthesising a click-event through the DOM just to make the same thing happen.

I'm working on this component at the moment, so I might clean it up while I'm at it. Unless you tried that way already and there was some specific reason it didn't work?

Reply:
I just did what I was told 😄 I think you can improve it the way you believe is better.

Reply:
Fixed in #1451.